examples/indexer/src/indexer.ts

```diff
@@ -31,10 +32,14 @@ export function createIndexerConfig(streamUrl: string) {
         },
       ],
     },
-    transform({ header, logs }) {
+    // biome-ignore lint/suspicious/noExplicitAny: <explanation>
+    sink: sqlite({ dbPath: "sqlite_test.db", tableName: "test" }) as any,
```

Comment:
Does putting `sink` after all other properties (including hooks) help with this issue?

Reply:
I'll check how to improve the types so that it's not needed because it's annoying.

Reply:
> Does putting `sink` after all other properties (including hooks) help with this issue?

Nope! didn't work. looks like the unknown type from `SqliteSink` is overriding the type defined in `IndexerConfig`

Reply:
Interesting. For now let's keep it like that and fix it later.